src/status_im/contexts/onboarding/identifiers/view.cljs

```diff
@@ -1,71 +0,0 @@
-(ns status-im.contexts.onboarding.identifiers.view
```

Comment:
@jo-mut - I don't think we need to remove the screen. 
At some point we are likely to show this elsewhere in the application, e,g when a user first clicks onto their profile screen. 
Let's keep the code??
cc @ilmotta @xAlisher

Reply:
I was under the impression that we are not going to need this anymore but I totally get it and agree with you @J-Son89 . I revert some of these changes to still retain the screen

Reply:
I believe we'll continue to keep the on-boarding slim and not add unnecessary steps, so I don't see this step coming back to the on-boarding funnel.

But if we really plan on using this step sounds good to keep it, but otherwise we just have git history and the GitHub PR to resurrect dead code.

Another idea is to feature flag, so that it's not just dead code lying around.

Reply:
yes this screen should be removed from onboarding, but please check with design. one idea presented to me before was to have this screen as a once off elsewhere in the app.

Reply:
It does look like a screen that may at somepoint be needed else where in the app. I thought git history would be okay in this context however to  just update the navigation makes more sense. I will check with design also